print-cover.html

```diff
@@ -253,7 +253,7 @@ <h1 class="title">Standard for Public Code</h1>
       <img src="assets/hands-shake.svg" alt="hands shaking">
 
       <div id="version-info">
-        <p><strong>Draft</strong><br/>Version {{ site.version }}</p>
+        <p><strong>Beta</strong><br/>Version {{ site.version }}</p>
```

Comment:
In the context of a document, the word "Draft" is more meaningful than "Beta" for most readers.
As such, I'd suggest we continue to say "Draft" in bold and perhaps add "beta" after the version.